packages/block-library/src/navigation-link/edit.js

```diff
@@ -278,6 +279,7 @@ function NavigationLinkEdit( {
 											return escape( normalizedURL );
 										} )(),
 										opensInNewTab: newOpensInNewTab,
+										type,
```

Comment:
Where is this defined in the attribute schema of the block?

Reply:
[Here](https://github.com/WordPress/gutenberg/blob/8e4d714c9da0ec979cbb14a16ee47d1a1fba0293/packages/block-library/src/navigation-link/block.json#L12) we have type.